community/giblib/sources

```diff
@@ -0,0 +1 @@
+https://kent.dl.sourceforge.net/project/slackbuildsdirectlinks/giblib/giblib-1.2.4.tar.gz
```

Comment:
This source doesn't point to upstream? Unless upstream is SlackWare(?)

Reply:
-.-' argh, my error, i was in two things simultaneously ....  I proceed to the changes

Reply:
original project page avaiable at http://freshmeat.sourceforge.net/projects/giblib but source download not avaiable. For this i add it in my github page.